cmd/forwarder/fixer.go

```diff
@@ -45,8 +45,15 @@ func fix(r io.Reader, remoteAddr string, logplexDrainToken string) ([]byte, erro
 		messageWriter.WriteString(" [origin ip=\"")
 		messageWriter.WriteString(remoteAddr)
 		messageWriter.WriteString("\"]")
-
 		b := lp.Bytes()
+
+		var replace bool
+
+		if authUser != "" && bytes.HasSuffix(b, []byte("\n")) {
```

Comment:
@jmervine under what circumstances can authUser actually be empty here? I think there should always be an authUser, which would make the `authUser != ""` check redundant.

Reply:
Configuration isn't passed to the fixer method. So when the env var isn't set, or is set to false, this will be `""`. It should be the only case where it's `""` as well, otherwise it'll be set to `"none"` if on and missing.